includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -15333,6 +15209,31 @@ class AMP_Allowed_Tags_Generated {
 					'unique' => true,
 				),
 			),
+			array(
+				'attr_spec_list' => array(
+					'amp-custom-length-check' => array(
+						'dispatch_key' => 1,
+						'mandatory' => true,
+						'value' => array(
+							'',
+						),
+					),
+					'nonce' => array(),
+					'type' => array(
+						'value_casei' => array(
+							'text/css',
+						),
+					),
+				),
+				'cdata' => array(
+					'max_bytes' => -1,
+				),
+				'tag_spec' => array(
+					'mandatory_parent' => 'head',
+					'spec_name' => 'style amp-custom-length-check',
+					'unique' => true,
+				),
+			),
```

Comment:
I'm not sure that this should be included. It comes from https://github.com/ampproject/amphtml/blob/41441daeabef58ffc753188914644f2893a7127a/validator/validator-main.protoascii#L1119-L1152

Note the comment:

```
# This variant of <style amp-custom> is designed to help developers see how
# close they are to the byte limit, even if the page is passing. By changing
# the attribute to <style amp-custom-length-check>, this tagspec will be used
# to match errors. It cannot pass however as it has a cdata.max_bytes set to
# -1. As a result, it will always fail and report the length found in the
# tag's cdata. This is a small hack, and may be deleted in the future if there
# are problems maintaining it. Deleting this tagspec will of course never
# break any valid AMP documents as if it matches, the page will be invalid.
```

I'm not entirely sure how this is supposed to work. In any case, it won't ever match due to a negative `max_bytes`. Also, we have better facilities for warning users that they are approaching the limit, including the CSS manifest comment presently which we'll integrate into the validated URL screen as of https://github.com/ampproject/amp-wp/issues/2169.